dbt_sugar/core/task/doc.py

```diff
@@ -56,10 +61,126 @@ def run(self) -> int:
+            {
+                "type": "confirm",
+                "name": "wants_to_document_model",
+                "message": f"Do you want to change the {model_name} model description?",
+                "default": True,
+            },
+            {
+                "type": "text",
+                "name": "model_description",
+                "message": "Please write down your description:",
+            },
+        ]
+        user_input = UserInputCollector("model", model_doc_payload).collect()
+        if user_input.get("model_description", None):
+            for model in content.get("models", []):
+                if model["name"] == model_name:
+                    model["description"] = user_input["model_description"]
+        return content
+
+    def find_model_in_dbt(self, model_name: str) -> Tuple[Optional[Path], bool]:
+        """
+        Method to find a model name in the dbt project.
+
+            - If we find the sql of the model but there is no schema we return the Path
+            and False (to create the schema).
+            - If we find the sql of the model and there is schema we return the Path and True.
+
+        Args:
+            model_name (str): model name to find in the dbt project.
+
+        Returns:
+            Tuple[Optional[Path], bool]: Optional path of the sql model if found
+            and boolean indicating whether the schema.yml exists.
+        """
+        for root, _, files in os.walk(self.repository_path):
+            files = [f for f in files if not re.match(EXCLUDE_TARGET_FILES_PATTERN, f)]
+            for file in files:
+                if file == f"{model_name}.sql":
+                    path_file = Path(os.path.join(root, "schema.yml"))
+                    if path_file.is_file():
+                        return path_file, True
+                    else:
+                        return path_file, False
+        return None, False
+
+    def orchestrate_model_documentation(self, model_name: str, columns_sql: List[str]) -> int:
+        """
+        Orchestrator to fully document a model will:
 
-        self.process_model(model, columns_sql)
+            - Create or update the columns found the database table into the schema.yml.
+            - Gives the user the posibility to change the model description.
+            - Gives the user the posibility to document the undocumented columns.
+            - Updates all the new columns definitions in all dbt.
+
+        Args:
+            model_name (str): Name of the model to document.
+            columns_sql (List[str]): Columns names that the model have in the database.
+
+        Returns:
+            int: with the status of the execution. 1 for fail, and 0 for ok!
+        """
+        content = None
+        path, exists = self.find_model_in_dbt(model_name)
+        if not path:
+            logger.info("Not able to find the model in the repository.")
+            return 1
+        if exists:
```

Comment:
what is the difference between path and exists? I imagine that if `not path` is true then `exists is false right?